spec/graphql/mutations/customers/update_spec.rb

```diff
@@ -46,6 +46,7 @@
           currency: 'EUR',
           providerCustomer: {
             providerCustomerId: 'cu_12345',
+            providerPaymentMethods: ['card', 'sepa_debit'],
```

Comment:
Style/WordArray: Use `%w` or `%W` for an array of words. (https://rubystyle.guide#percent-w)